README.md

```diff
@@ -1,6 +1,13 @@
-# React Flagpack
+# Flagpack
```

Comment:
We should consider use [Shields](https://shields.io) at the top of the project `readme.md`. It's used in a lot of Open Source projects and is somewhat the standard. It gives the user/contributer a clear overview of the current status of the project, but these icons can be set up pretty dynamically.

Reply:
Like it! I'll add it to Jira as new task.